lib/middleware/karma.js

```diff
@@ -114,8 +114,10 @@ var createKarmaMiddleware = function (
       })
     }
 
-    // serve karma.js
-    if (requestUrl === '/karma.js') {
+    // serve karma.js, context.js, and context-debug.js
+    var jsFiles = ['/karma.js', '/context.js', '/context-debug.js']
```

Comment:
I'm confused. Do you mean `requestUrl.match(/(karma|context|context-debug)\/.js` or to always do replacement (effectively an `if true`)?